src/prelude.rs

```diff
@@ -0,0 +1,8 @@
+// Copyright 2015-2016, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+//! Traits and essential types inteded for blanket imports.
+
+pub use auto::traits::*;
+
```

Comment:
`pub use ffi::PangoMatrix as PangoMatrix;` 
Or Matrix trait will be unusable without adding pango-sys dependence